### ISSUE TITLE
should keep unicode space char

### ISSUE THREAD

**Issue by jaylecn** [svg/svgo#1150]
input

```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400" height="500">
<text fill="#000000" stroke="none" font-family="sans-serif" font-size="10px" font-style="normal" font-weight="normal" text-decoration="normal" x="20" y="20" text-anchor="start" dominant-baseline="alphabetic">
test&#160;spaces&#160;&#160;&#160;😑&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;end
</text></svg>
```

output

```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400" height="500"><text fill="#000000" stroke="none" font-family="sans-serif" font-size="10px" font-style="normal" font-weight="normal" text-decoration="normal" x="20" y="20" text-anchor="start" dominant-baseline="alphabetic">test spaces 😑 end</text></svg>
```

i want keep spaces, but even i use unicode space char, it's still removed,  bug ?

**Comment by jaylecn** [comment]
it's latest commit on 2017 😶 any solution?  fix sax-js ?

**Comment by KrasnayaPloshchad** [comment]
Sax-js is no longer maintained after version 1.2.7 released, but I discovered a fork of this parser in their issues. This is under active maintaince, but has some differences than original one.
https://github.com/lddubeau/saxes

**Comment by strarsis** [comment]
@KrasnayaPloshchad: So it wouldn't be that hard to replace sax.js with saxes?
When it is more or less just a drop-in replacement I could give it a try.

**Comment by KrasnayaPloshchad** [comment]
I think yes.

**Comment by SethFalco** [comment]
This was not an issue in sax, and rather was a configuration issue on our end. A contributor corrected this for us, and the fix was released in SVGO v2.1.0:

* https://github.com/svg/svgo/commit/9f084cc0865dc2dd5fa867afbbe8e10647757a6f#diff-4a0b83ac4f1aaf48931fcdbfc22b5a79bd10cc3a414e2b4a6ce22b622c4cf2feR13
* https://github.com/svg/svgo/pull/1220

The optimizing the SVG works as expected in SVGO v3.2.0.
